pkg/util/alibaba/alibaba.go

```diff
@@ -20,38 +20,48 @@ var (
 
 // GetHostAlias returns the VM ID from the Alibaba Metadata api
 func GetHostAlias() (string, error) {
-	res, err := getResponse(metadataURL + "/latest/meta-data/instance-id")
+	res, err := getResponseWithMaxLength(metadataURL+"/latest/meta-data/instance-id", 255)
```

Comment:
Could we move the `255` to the config package maybe ? That would give us some flexibility in case this cause major issue for a some customers and would avoid duplicating the hardcoded value across all cloud providers.
 
I don't think we need to expose it in the datadog.yaml.

Reply:
Added it to the config.